R/get_paths.R

```diff
@@ -331,10 +331,14 @@ get_sorted <- function(x) {
 
   # Some vectors to track results
   out <- rep(0, n)
```

Comment:
to_visti <- out <- rep(0, nrow(x)) : might be cleaner and easier to follow? then remove 322, and 341